react/week3/to-do-app/src/ToDo.js

```diff
@@ -0,0 +1,49 @@
+import React, { useState } from 'react';
+import Border from './Border';
+
+function ToDo(props){
+    const [enabled, setEnabled] = useState(false);
+    const [editMode, setEditMode] = useState(false);
+    const [validUpdatedDesc, setValidUpdatedDesc] = useState(true);
+      const handleEdit = () => {
+        setEditMode(true);
+      }
+      const handleUpdate = (id) => {
+          const updatedDesc=document.getElementById('input'+id).value;
```

Comment:
Kind of the same thing as I commented on above. It should not really be necessary to use getElementById, there are better ways to do it when we are working with React.